docs/translations/api-docs/card/card-de.json

```diff
@@ -3,7 +3,7 @@
   "propDescriptions": {
     "children": "The content of the component.",
     "classes": "Override or extend the styles applied to the component. See <a href=\"#css\">CSS API</a> below for more details.",
-    "raised": "If <code>true</code>, the card will use raised styling."
+    "raised": "If <code>true</code>, the card will use raised styling. Has no effect when used with <code>variant={'outlined'}</code>."
```

Comment:
Did you manually change it? If you did, please revert, it's managed in Crowdin

Reply:
oh good, i thought that was a lot to manage 👍